.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "ignore": [
+    "**/dist/ui-kit.esm.js"
```

Comment:
Should we just ignore all if `dist` just in case we ever rename the file? I doubt we'd ever need to compile any of `dist` anyways? And maybe a code comment why we need `.babelrc` and why we ignore it?

Reply:
yeah, that makes sense, especially since `react-percy` excludes `node_modules` from their babel config anyways. 👍

Reply:
I updated it but I left out a comment. Sometimes babelrc is treated as a JSON and sometimes not, so I don't think it's worth chancing it...